x/delegation/keeper/delegation_op_test.go

```diff
@@ -145,13 +147,15 @@ func (suite *DelegationTestSuite) TestUndelegateFrom() {
 
 	operatorState, err := suite.App.AssetsKeeper.GetOperatorSpecifiedAssetInfo(suite.Ctx, delegationEvent.OperatorAddress, assetID)
 	suite.NoError(err)
-	suite.Equal(types.OperatorAssetInfo{
+	ok := suite.Equal(types.OperatorAssetInfo{
 		TotalAmount:         sdkmath.NewInt(0),
 		WaitUnbondingAmount: delegationEvent.OpAmount,
 		TotalShare:          sdkmath.LegacyNewDec(0),
 		OperatorShare:       sdkmath.LegacyNewDec(0),
 	}, *operatorState)
-
+	if !ok {
+		log.Error("not ok")
```

Comment:
Instead of `Equal`, maybe `Assert` ?

Reply:
Is this modification being made because the test fails without any related code changes? If so, I can record this issue and look into the cause when I have time later.